wds-cmb2-date-range-field.php

```diff
@@ -192,6 +193,51 @@ function sanitize( $override_value, $value ) {
 		return $value;
 
 	}
+	
+	/**
+	 * Save the start date and end date into separate fields if the argugment split_date_range is true
+	 *
+	 * @param string            $field_id The current field id paramater.
+	 * @param bool              $updated  Whether the metadata update action occurred.
+	 * @param string            $action   Action performed. Could be "repeatable", "updated", or "removed".
+	 * @param CMB2_Field object $field    This field object
+	 */
+	function save_split_date_range( $field_id, $updated, $action, $cmb2_field ) {
+
+		if ( ! $updated || $action == 'repeatable' ) {
+			return;
+		}
+
+		$split_values = $cmb2_field->args( 'split_date_range' );
+
+		if ( $split_values === true ) {
```

Comment:
Truthy check is good enough.
```suggestion
		if ( $cmb2_field->args( 'split_date_range' ) ) {
```